crates/ev-prover/src/commands/command.rs

```diff
@@ -108,7 +108,7 @@ pub async fn create_zkism() -> Result<()> {
     let pub_key = get_sequencer_pubkey(sequencer_rpc_url).await?;
 
     let ev_hyperlane_vkey = fs::read("testdata/vkeys/ev-hyperlane-vkey-hash")?;
-    let ev_combined_vkey = fs::read("testdata/vkeys/ev-range-exec-vkey-hash")?;
+    let ev_combined_vkey = fs::read("testdata/vkeys/ev-combined-vkey-hash")?;
```

Comment:
We should just setup a cpu prover in the create cmd and use it to `setup(ELF)` for these. 
Using the local file paths is brittle because it expects the binary to be invoked from a location which has `testdata/vkeys/...` on their machine. 

We should also load the config from yaml in this function rather than using env variables.

I will create an issue and open a PR